### PR TITLE
Updated polish translation

### PR DESCRIPTION
**Pull request by nikodemak**
Not tested in any way. description in first commit

### CODE REVIEW

File: imports/i18n/pl/language.js [hunk 1]
```diff
@@ -8,25 +8,31 @@ const language = {
...
+      en: 'Angielski',
+	  'en-AU': 'Angielski (Australia)',
       fi: 'Fiński',
       fr: 'Francuski',
       hu: 'Węgierski',
-      it: 'Włoski (częsciowo)',
+      it: 'Włoski (częśćiowo)',
       pl: 'Polski',
       pt: 'Portugalski',
       ru: 'Rosyjski'
```
Comment (MarvinZeising) [inline imports/i18n/pl/language.js]: comma missing at the end of the line

File: imports/i18n/pl/language.js [hunk 2]
```diff
@@ -8,25 +8,31 @@ const language = {
...
+	  'zh-TW': 'Chinese (Tradycyjny) (częśćiowo)',
```
Comment (MarvinZeising) [inline imports/i18n/pl/language.js]: indentation seems to be incorrect (two spaces missing?)

File: imports/i18n/pl/vessels.js [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+      deleteConfirmation: 'Czy na pewno chcesz usunąć tą wizytę?',
+      person: 'Głosiciel',
+      email: 'Adres E-mail głosiciela',
+      phone: 'Numer telefonu głosiciela',
+      isUserVisible: 'Uczynić twoje dane kontaktowe widoczne dla innych głosicieli?',
+      date: 'Data',
+      dateNext: 'Następna wizyta najlepiej po',
+      harbor: 'Port',
+      harborId: 'Port',
+      harborIdValues: {
```
Comment (MarvinZeising) [inline imports/i18n/pl/vessels.js]: this object doesn't have to be provided if nothing is different from the english default translation

File: imports/i18n/pl/framework.js [hunk 4]
```diff
@@ -22,9 +22,9 @@ const framework = {
     no: 'Nie'
   },
   dateFormat: {
-    date: 'DD.MM.YYYY',
-    dateAndTime: 'DD.MM.YYYY HH:mm',
-    time: 'HH:mm'
+    date: 'DD.MM.RRRR',
```
Comment (MarvinZeising) [inline imports/i18n/pl/framework.js]: what is RRRR supposed to mean?
I can't find it under the format identifiers here: https://momentjs.com/docs/#/parsing/string-format/
I guess it should stay YYYY, maybe?

File: imports/i18n/pl/framework.js [hunk 5]
```diff
@@ -22,9 +22,9 @@ const framework = {
...
+    dateAndTime: 'DD.MM.RRRR GG:mm',
+    time: 'GG:mm'
```
Comment (MarvinZeising) [inline imports/i18n/pl/framework.js]: according to https://momentjs.com/docs/#/parsing/string-format/, GG would display the ISO 2 digit week year (in 2018 this means "18")
I'm not sure if this is really what you want to achieve...

File: imports/i18n/pl/vessels.js [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+  },
+  search: {
+    placeholder: 'Nazwa statku, Sygnał Wywoławczy, ENI, IMO albo MMSI'
+  },
+  details: {
+    sections: {
+      identification: 'Dane Identyfikacyjne',
+      visit: 'Dane wizyty'
+    },
+    dateFormat: 'MM/DD/RRRR'
```
Comment (MarvinZeising) [inline imports/i18n/pl/vessels.js]: same here

File: imports/i18n/pl/vessels.js [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+  },
+  visit: {
+    details: {
+      sections: {
+        main: 'Dane wizyty',
+        language: 'Language data',
+        option: 'Opcje'
+      },
+      dateFormat: 'MM/DD/RRRR'
```
Comment (MarvinZeising) [inline imports/i18n/pl/vessels.js]: same here

File: imports/i18n/pl/vessels.js (hunk 3 above)
Comment (nikodemak) [reply imports/i18n/pl/vessels.js]: Port happens to mean the same in polish and in english, but harbor does not

File: imports/i18n/pl/language.js (hunk 2 above)
Comment (nikodemak) [reply imports/i18n/pl/language.js]: this is taken directly from english translation (i forgot to translate "chinese" though)

Comment (MarvinZeising) [reply imports/i18n/pl/language.js]: The english translation has 6 spaces at the beginning of the line.
This polish one has 1 tab and 2 spaces. Sadly, this makes a difference. I'm sorry.
Has to be 6 spaces

File: imports/i18n/pl/vessels.js (hunk 3 above)
Comment (MarvinZeising) [reply imports/i18n/pl/vessels.js]: Sorry, I meant that the whole harborIdValues: { ... } thing can be omitted (deleted)

**Comment by nikodemak** [comment]
its good now?

**Comment by MarvinZeising** [comment]
Okay, done. Thank you very much @nikodemak! :-)

**Comment by nikodemak** [comment]
NP
i would suggest testing it though, something might be still wrong

**Pull request merged**